yb_migrate/src/migration/oracle.go

```diff
@@ -193,28 +194,21 @@ func Ora2PgExportDataOffline(ctx context.Context, source *srcdb.Source, exportDi
 		source.NumConnections, projectDirPath, configFilePath)
 
 	//Exporting all the tables in the schema
-	exportDataCommand := exec.Command("/bin/bash", "-c", exportDataCommandString)
+	exportDataCommand := exec.CommandContext(ctx, "/bin/bash", "-c", exportDataCommandString)
 	log.Infof("Executing command: %s", exportDataCommandString)
+	var outbuf bytes.Buffer
+	var errbuf bytes.Buffer
 
-	stdOutFile, err := os.OpenFile(exportDir+"/temp/export-data-stdout", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
-	if err != nil {
-		utils.ErrExit("Error while opening export data output file: %v", err)
-	}
-	defer stdOutFile.Close()
+	exportDataCommand.Stdout = &outbuf
+	exportDataCommand.Stderr = &errbuf
 
-	stdErrFile, err := os.OpenFile(exportDir+"/temp/export-data-stderr", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
-	if err != nil {
-		utils.ErrExit("Error while opening export data error file: %v", err)
-	}
-	defer stdErrFile.Close()
-
-	exportDataCommand.Stdout = stdOutFile
-	exportDataCommand.Stderr = stdErrFile
-
-	err = exportDataCommand.Start()
+	err := exportDataCommand.Start()
 	fmt.Println("starting ora2pg for data export...")
+	if outbuf.String() != "" {
+		log.Infof("ora2pg STDOUT: %s", outbuf.String())
+	}
 	if err != nil {
-		utils.ErrExit("Error while starting ora2pg for data export: %v", err)
+		utils.ErrExit("Error while starting ora2pg for data export: %v\n%s", err, errbuf.String())
```

Comment:
You will also need to log the `errbuf` if the `exportDataCommand.Wait()` returns error on line 216.

Reply:
+1